tests/instruments/attocube/test_anc300.py

```diff
@@ -0,0 +1,61 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2022 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+import pytest
+
+from pymeasure.instruments.attocube import ANC300Controller
+from pymeasure.instruments.attocube.adapters import Mock_TelnetAdapter, AttocubeConsoleAdapter
+
+#############################################################################
+# In order to run the tests, you have to change the superclass of the adapter
+# from TelnetAdapter to Mock_TelnetAdapter
```

Comment:
can't you say here 
```
pymeasure.instruments.attocube.adapters.AttocubeConsoleAdapter = Mock_TelnetAdapter
```
and then instantiate an `ANC300Controller`

Reply:
there's even a mock/monkeypatch context manager in the stdlib to do this "safely" (i.e. undo it after leaving the scope), but the details escape me right now.

Reply:
I tried it with monkeypatch, but was not able to get it to work, therefore I did this dirty solution.

I'll try it with that suggestion.

Reply:
I did stuff like that with `print` before 😅 you gotta love Python

Reply:
Oh. I do not replace the AttocubeAdapter with the Mock_TelnetAdapter, but I replace the TelnetAdapter (parent class of AttocubeAdapter) with the Mock_Telnet. In other words, I want to replace the parent class of the AttocubeAdapter.

That is more difficult to do.

Reply:
I just kept the (for me successfully run) test parameters and mark them to fail. After the adapter rework, they should pass.